### ISSUE TITLE
Reactive attributes: a_disabled cannot be reactive

### ISSUE THREAD

**Issue by vouillon** [ocsigen/eliom#225]
The `a_disabled` attribute has a single value `` `Disabled``, so there is no way to make it reactive.

**Comment by vasilisp** [comment]
We have `filter_attrib` for cases like this. We probably need to separate the "unary" attributes (like `a_disabled`) in the TyXML functors, to avoid dealing with constant signals.

**Comment by Drup** [comment]
Yes, filter_attrib is the right solution. Unfortunatly it's not in tyxml directly, because it needs rather intricate knowledge of the actual react implementation.

I'm of the opinion that we should remove the argument for these attributes and make them really constant.
